### ISSUE TITLE
[FE8] Early game shamans unusable

### ISSUE THREAD

**Issue by vklein52** [lushen124/Universal-FE-Randomizer#321]
Since the shops don’t start selling Flux until around chapter 12 or 13, early game shamans become completely useless almost immediately. 

I think either adding Flux to an early chapter’s overworld shop or forcing all user shamans to have multiple flux tomes is the easiest and most logical solution to this problem.

**Comment by vklein52** [comment]
It seems like it actually is available after chapter 9 upon further inspection, but adding Flux to Serafew’s overworld shop may be worthwhile still. Or just hardcoding multiple tomes still. :)

**Comment by lushen124** [comment]
That is true. I'm open to adding it to earlier shops, though I think this is probably more of a pressing issue for FE7 and FE6, which don't have an easily accessible shop to visit at any point. And Dark tomes don't show up until your first vanilla Shaman (Raigh in FE6 and Canas in FE7. For FE7 it's not too bad, that's like 7 chapters off of a Flux. But for FE6, it can be crippling because you have to go 13 chapters without a way of replenishing. 

One idea I had to fix the issue more generally was to have more adaptive shops that would adjust to your expected party in the early game. It'd phase out by mid-game, but at least the early shops would have the basics your party could use. Would have to test this out a bit more to see if it's feasible.
